src/osd/PG.cc

```diff
@@ -5975,6 +5978,18 @@ boost::statechart::result PG::RecoveryState::Peering::react(const QueryState& q)
   }
   q.f->close_section();
 
+  if (history_les_bound) {
+    q.f->open_object_section("peering_blocked_by_options");
```

Comment:
Probably best not to mention the option (it's kludgy anyway).  Maybe make it say:

    peering_blocked_by_details : [
        {
            detail: peering_blocked_by_history_les_bound
        }
    ]
}